src/ui/settings/colorMode.js

```diff
@@ -18,10 +20,28 @@ const ColorModeSettings = () => {
     <SettingsSection title={t('settings-theme-appearance')}>
       <Button
         onClick={() => setColorMode(colorMode === 'dark' ? 'light' : 'dark')}>
-        {colorMode === 'dark' ? light + ' ☀️' : dark + ' 🌙'}
+        {colorMode === 'dark' ? light : dark}
+        <ThemeIcon/>
       </Button>
     </SettingsSection>
   );
 };
 
-export default ColorModeSettings;
\ No newline at end of file
+const ThemeIcon = () => {
+  const [colorMode] = useColorMode();
+
+  return(
+    <div
+      sx={{
+        color: '#ffd983',
```

Comment:
That color unfortunately has fairly low contrast with the green of the button. Maybe Just make it white or black?